src/Components/PostsList/PostsList.test.js

```diff
@@ -0,0 +1,40 @@
+import { render, screen } from "../../Utils/test-utils";
+import { PostsList } from "./index";
+import { mockedPosts } from "./Posts_mocks";
+
+describe("PostsList", () => {
+  describe("When fetching data", () => {
+    it("Renders the loading message ", () => {
```

Comment:
![Screenshot from 2022-04-08 09-28-00](https://user-images.githubusercontent.com/54550926/162435541-b85a4248-f424-4fe2-a17d-f569a5e17e4b.png)

```suggestion
    it("Renders the loading message", () => {
```